epicbox-hyperskill/node/Dockerfile

```
@@ -1,8 +1,8 @@
-FROM node:15.10-stretch-slim
+FROM node:18-bullseye-slim
 
-RUN apt-get update && \
-    apt-get install -y python3 && \
-    apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
+RUN apt-get update
+RUN apt-get install -y python3
+RUN apt-get install -y gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
```

This increases the size of the docker image.
```suggestion
RUN apt-get update && \
    apt-get install -y python3 \
    gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 \
```